pkg/executor/recover_test.go

```diff
@@ -555,7 +555,7 @@ func TestFlashbackSchema(t *testing.T) {
 		require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/pkg/meta/autoid/mockAutoIDChange"))
 	}()
 
-	store := testkit.CreateMockStore(t)
+	store := testkit.CreateMockStore(t, mockstore.WithStoreType(mockstore.EmbedUnistore))
```

Comment:
Do not use image here, there may be some old ddl jobs causing get a wrong error in the tests.